addons/radios/functions/fnc_lowerHeadset.sqf

```diff
@@ -0,0 +1,43 @@
+/*
+Author: Grey
+
+Adjusts SW & LR radio volume
+
+Arguments:
+0: unit <OBJECT>
+
+Return value:
+None
+*/
+#include "script_component.hpp"
+
+private "_haveSW,_haveLR,_radioSW,_radioLR,_isLowered";
+
+_haveSW = call TFAR_fnc_haveSWRadio;
+_haveLR = call TFAR_fnc_haveLRRadio;
+
+_radioSW = call TFAR_fnc_activeSwRadio;
+_radioLR = call TFAR_fnc_activeLrRadio;
+
+_isLowered = ACE_player getVariable [QGVAR(headsetIsLowered),false];
+if(_isLowered) then{
```

Comment:
Since you do not use `_isLowered` anywhere else, you can replace the lines 22 and 23 by:
```sqf
if (ACE_player getVariable [QGVAR(headsetIsLowered),false]) then {
...
```